### PR TITLE
Import type vertices

### PR DESCRIPTION
**Pull request by jayrbolton**

### CODE REVIEW

File: src/test/test_releng_import_object.py [hunk 1]
```diff
@@ -218,6 +218,10 @@ def test_genome(self):
             'ws_object_version/6:7:8',  # from
             'ws_type_version/KBaseGenomes.Genome-15.1'  # to
         )
+        # Check for type vertices
+        self.assertTrue(get_re_doc('ws_type_version', obj_type))
```
Comment (MrCreosote) [inline src/test/test_releng_import_object.py]: I'd check the structure rather than they just exist

File: src/utils/re_client.py [hunk 2]
```diff
@@ -130,6 +130,8 @@ def save(coll_name, docs, on_duplicate='update', display_errors=False):
             'error'.
         display_errors - include error information if insert errors occur.
     """
+    if not isinstance(docs, list) and isinstance(docs, dict):
```
Comment (MrCreosote) [inline src/utils/re_client.py]: Update docstring

Comment (MrCreosote) [inline src/utils/re_client.py]: Isn't the first check redundant? If it's a dict it's not a list

File: src/test/test_releng_import_object.py [hunk 3]
```diff
@@ -218,6 +218,10 @@ def test_genome(self):
...
+        self.assertTrue(get_re_doc('ws_type', f"{type_module}.{type_name}"))
+        self.assertTrue(get_re_doc('ws_type_module', type_module))
```
Comment (MrCreosote) [inline src/test/test_releng_import_object.py]: I suppose there's no reasonable way to test the lru cache

File: src/test/test_releng_import_object.py (hunk 1 above)
Comment (jayrbolton) [reply src/test/test_releng_import_object.py]: Always calling out my laziness. Just updated to use assertDictContainsSubset

File: src/utils/re_client.py (hunk 2 above)
Comment (jayrbolton) [reply src/utils/re_client.py]: Updated

Comment (jayrbolton) [reply src/utils/re_client.py]: Updated

**Pull request merged**